package/yast2-storage-ng.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Wed Jul 12 15:37:16 UTC 2017 - ancor@suse.com
+
+- Replaced the original prototype of the expert partitioner with
+  the one coming from yast2-partitioner.
+- Removed Y2Storage::DevicesLists and not longer needed modules
+  from Y2Storage::Refinements
```

Comment:
well, it makes sense for me, as it is public API and y2-partitioner is not the only user of API.
another option for me is to not mention it in changelog and use instead `@since` tag so it is easy to see which version add it. And same we should do for method removal, so lifetime of API is clear